src/main.cpp

```diff
@@ -2382,7 +2382,7 @@ bool ConnectBlock(const CBlock &block,
     bool fScriptChecks = true;
     if (pindexBestHeader)
         fScriptChecks = !fCheckpointsEnabled || block.nTime > timeBarrier ||
-                        pindex->nHeight > pindexBestHeader->nHeight - (144 * checkScriptDays.value);
```

Comment:
I'll change pindex->nHeight to a uint32...it's rather odd that pindex->nHeights are int's and pindexBestHeader->nheights are uints...i think we could change that in the future but it's for another PR...I just threw this in here because of a nuisance compiler warning anyway...i don't want to spend a lot of time on it.